packages/orbit-components/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -16,7 +16,7 @@ const meta: Meta<typeof Checkbox> = {
   parameters: {
     info: "Additionally you can add tooltip to this component.",
     controls: {
-      exclude: ["onChange"],
+      exclude: ["onChange", "defaultChecked"],
```

Comment:
I would add a separate story for the variant with `defaultChecked` prop, it could be called `Uncontrolled`. We have this type of story in Textarea or Card. WDYT?

Reply:
Yes, we can have uncontrolled stories here as well. Will add